pkg/dhclient/dhclient.go

```diff
@@ -204,7 +204,7 @@ func lease6(ctx context.Context, iface netlink.Link, c Config) (Lease, error) {
 	}
 
 	log.Printf("Attempting to get DHCPv6 lease on %s", iface.Attrs().Name)
-	p, err := client.RapidSolicit(ctx, dhcpv6.WithNetboot)
+	p, err := client.Solicit(ctx, dhcpv6.WithNetboot)
```

Comment:
If you want to make this change, it's gotta be a different PR, I don't think this belongs in here

Reply:
accident :-)

Reply:
THIS TIME FOR SURE .... I MEAN IT ....